packages/itwinui-react/src/core/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -87,9 +104,13 @@ export const ToggleSwitch = React.forwardRef((props, ref) => {
           'iui-label-on-left': label && labelPosition === 'left',
         },
         className,
+        wrapperProps?.className,
```

Comment:
It feels strange to have both `className` and `wrapperProps.className` applied on the same element.

We could do something similar to what we did with ARIA props in `Table`, i.e. the presence of `wrapperProps` implies that `className` should be applied on `<input>`.

Reply:
…that is the opposite of what I meant.

`wrapperProps` is a standalone feature that should be added regardless of our future plans, whereas `future.consistentPropsSpread` might require more discussion and we may not release it in the next minor.